cmd/skopeo/utils.go

```diff
@@ -18,6 +18,7 @@ func contextFromGlobalOptions(c *cli.Context, flagPrefix string) (*types.SystemC
 		DockerInsecureSkipTLSVerify: !c.GlobalBoolT("tls-verify"),
 		OSTreeTmpDirPath:            c.String(flagPrefix + "ostree-tmp-dir"),
 		OCISharedBlobDirPath:        c.String(flagPrefix + "shared-blob-dir"),
+		ForceCompress:               c.Bool("compress"),
```

Comment:
(Having a src/destination-specific option without `flagPrefix` seems unexpected. Non-blocking I guess, the alternative is not _clearly_ better, but I’d be a bit happier with a `flagPrefix+"compress"`, i.e. `--dest-compress`, for consistency.)

Reply:
fixed

Reply:
The documentation and completions need to be updated as well.

Reply:
updated docs and completions.